pyaedt/edb_core/edb_data/hfss_simulation_setup_data.py

```diff
@@ -0,0 +1,1700 @@
+
+    @property
+    def layer_snap_tol(self):
+        """Layer snap tool.
+
+        Returns
+        -------
+
+        """
+        return self._advanced_mesh_settings.LayerSnapTol
+
+    @layer_snap_tol.setter
+    def layer_snap_tol(self, value):
+        self._advanced_mesh_settings.LayerSnapTol = value
+        self._parent._update_setup()
+
+    @property
+    def mesh_display_attributes(self):
+        """Mesh display attributes.
+
+        Returns
+        -------
+
+        """
+        return self._advanced_mesh_settings.MeshDisplayAttributes
+
+    @mesh_display_attributes.setter
+    def mesh_display_attributes(self, value):
+        self._advanced_mesh_settings.MeshDisplayAttributes = value
+        self._parent._update_setup()
+
+    @property
+    def replace_3d_triangles(self):
+        """Replace 3d triangles.
+
+        Returns
+        -------
+        bool
+        """
+        return self._advanced_mesh_settings.Replace3DTriangles
+
+    @replace_3d_triangles.setter
+    def replace_3d_triangles(self, value):
+        self._advanced_mesh_settings.Replace3DTriangles = value
+        self._parent._update_setup()
+
+
+class CurveApproxSettings(object):
+    """Manages EDB methods for curve approx settings."""
+
+    def __init__(self, parent):
+        self._parent = parent
+
+    @property
+    def _curve_approx_settings(self):
+        return self._parent._edb_sim_setup_info.SimulationSettings.CurveApproxSettings
+
+    @property
+    def arc_angle(self):
+        return self._curve_approx_settings.ArcAngle
+
+    @arc_angle.setter
+    def arc_angle(self, value):
+        self._curve_approx_settings.ArcAngle = value
+        self._parent._update_setup()
+
+    @property
+    def arc_to_chord_error(self):
+        return self._curve_approx_settings.ArcToChordError
+
+    @arc_to_chord_error.setter
+    def arc_to_chord_error(self, value):
+        self._curve_approx_settings.ArcToChordError = value
+        self._parent._update_setup()
+
+    @property
+    def max_arc_points(self):
+        return self._curve_approx_settings.MaxArcPoints
+
+    @max_arc_points.setter
+    def max_arc_points(self, value):
+        self._curve_approx_settings.MaxArcPoints = value
+        self._parent._update_setup()
+
+    @property
+    def start_azimuth(self):
+        return self._curve_approx_settings.StartAzimuth
+
+    @start_azimuth.setter
+    def start_azimuth(self, value):
+        self._curve_approx_settings.StartAzimuth = value
+        self._parent._update_setup()
+
+    @property
+    def use_arc_to_chord_error(self):
+        return self._curve_approx_settings.UseArcToChordError
+
+    @use_arc_to_chord_error.setter
+    def use_arc_to_chord_error(self, value):
+        self._curve_approx_settings.UseArcToChordError = value
+        self._parent._update_setup()
+
+
+class DcrSettings(object):
+    """Manages EDB methods for dcr settings."""
```

Comment:
```suggestion
    """Manages EDB methods for DCR settings."""
```
Will a user know what DCR means?